### DISCUSSION TITLE
Emoji picker

### DISCUSSION THREAD

**Discussion by oemueller** [SchwarzIT/onyx#1733] [Ideas]
Would be (a) nice to have in Onyx: https://nolanlawson.github.io/emoji-picker-element/
as well 🙂

**Comment by mj-hof** [comment]
Hi @oemueller. Can you tell us a usecase behind this idea for enterprise applications? 🤷🏼‍♂️

**Reply by oemueller** [reply]
possible use cases could be:

- helpdesk chats
- feedback forms
- tagging tasks

**Comment by bergser** [comment]
it is also possible to use native emoji pickers:
- windows: `win` `+` `.` or `win` `+` `;`
- mac: `cmd` `+` `ctrl` `+` `space`

**Comment by larsrickert** [comment]
For now we plan to not implement a dedicated emoji picker component so for now you can use the native emoji pickers mentioned by https://github.com/SchwarzIT/onyx/discussions/1733#discussioncomment-14603081. However, we will take another look at emojis for our upcoming Text editor component (#4354)
